src/main/java/software/sigma/bu003/internship/coparts/client/technomir/entity/responce/SupplierInPartFromTehnomir.java

```diff
@@ -0,0 +1,24 @@
+package software.sigma.bu003.internship.coparts.client.technomir.entity.responce;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class SupplierInPartFromTehnomir {
```

Comment:
entity - this is a class with @Id they are not entities... rather responses

Reply:
I'm sure they will be entities, it's already in the next tickets